packages/grafana-ui/package.json

```diff
@@ -43,6 +43,11 @@
     "@types/react-table": "7.0.12",
     "@types/slate": "0.47.1",
     "@types/slate-react": "0.22.5",
+    "@visx/event": "^1.3.0",
+    "@visx/gradient": "^1.0.0",
+    "@visx/scale": "^1.4.0",
+    "@visx/shape": "^1.4.0",
+    "@visx/tooltip": "^1.3.0",
```

Comment:
Can you lock these to latest (no ^), having a dynamic patch version makes updating only specific packages tricky so we try to specify full version

Reply:
Sure thing